src/test/mempool_tests.cpp

```diff
@@ -203,7 +203,13 @@ BOOST_AUTO_TEST_CASE(MempoolIndexingTest)
 
     CTxMemPool::setEntries setAncestorsCalculated;
     std::string dummy;
-    BOOST_CHECK_EQUAL(pool.CalculateMemPoolAncestors(entry.Fee(2000000LL).FromTx(tx7), setAncestorsCalculated, 100, 1000000, 1000, 1000000, dummy), true);
+    CTxMemPool::Limits mempool_limits{
+        .ancestor_count = 100,
+        .ancestor_size_vbytes = 1'000'000,
+        .descendant_count = 1'000,
+        .descendant_size_vbytes = 1'000'000,
+    };
+    BOOST_CHECK_EQUAL(pool.CalculateMemPoolAncestors(entry.Fee(2'000'000LL).FromTx(tx7), setAncestorsCalculated, mempool_limits, dummy), true);
```

Comment:
I think this can be replaced with `CTxMemPool::Limits::NoLimits()`, actually. These numbers seem arbitrarily high and don't restrict anything, suggesting the intent of this test case is just to calculate the ancestors properly.